### DISCUSSION TITLE
Git 101: git@github.com: Permission denied (publickey)

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [pglombardo/PasswordPusher#922] [Q&A] [answered]
I am trying to install from source. When I run the "git clone git@github.com:pglombardo/PasswordPusher.git" command, the download fails with "git@github.com: Permission denied (publickey)" followed by "fatal: Could not read from remote repository".

What am I missing?

**Comment by pglombardo** [comment, marked as answer]
Hi @ZuluPilgrim - did you add your SSH key to your Github account?  If not, then use HTTP method instead:

`git clone https://github.com/pglombardo/PasswordPusher.git`

Some other general troubleshooting information is here:

* https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-denied-publickey
* https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Or as a last option, you could download a release zip file (at the bottom of this page):
https://github.com/pglombardo/PasswordPusher/releases/tag/v1.26.8

**Reply by ZuluPilgrim** [reply]
Many thanks that was it :)
